packages/fast-colors/src/rgb.ts

```diff
@@ -0,0 +1,90 @@
+
+export function hexToRgb(hex: string): string {
+  let processedHex: IRgbJSON;
+  processedHex = properRGBValues(hex);
```

Comment:
I'd suggest defining `processedHex` as a const and assigning it to the return value of the function here since we are changing it anywhere else. You could likely do the same with everything below.

Reply:
+1